File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const seen = new Map()

        for(let num of nums) {
            if(seen.has(num)) return true
            seen.set(num)
        }

        return false
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        var ans = new List<int>();

        for (var i = 0; i < nums.Length; i++) {
            ans.Add(nums[i]);
        }

        for (var i = 0; i < nums.Length; i++) {
            ans.Add(nums[i]);
        }

        return ans.ToArray();
    }
}